lua/luasnip/config.lua

```diff
@@ -2,6 +2,7 @@ local defaults = {
 	history = false,
 	updateevents = "InsertLeave",
 	store_selection_keys = nil, -- Supossed to be the same as the expand shortcut
+	store_selection_register = "l", --name of the register to use for capturing selections
```

Comment:
Good idea to make it configurable, I think 9 would be a saner default.

Reply:
I prefer named registers, as numbered ones get populated automatically.

Reply:
Tbh here I'd just go after what is least likely to erase text needed by the user and I think it's safe to assume that one is more likely to use/need eg. `l` than the text from 9 yanks/deletions/... ago

Reply:
That's why I made it configurable  :smile: . Using registers in plugins is always risky as there is always a possibility of someone using the same register for something else.

Reply:
Oh yeah, configurable is a must, no question, I'm just arguing that `9` is less likely to be actively used than any of the character-registers and should therefore be the default instead of `l`.